rdrf/rdrf/initial_data/reference_data.py

```diff
@@ -3,16 +3,30 @@
 Includes:
     - Address Types
 """
+from django.template.loader import get_template
 
+from rdrf.events.events import EventType
+from rdrf.models.definition.models import EmailTemplate
 from registry.patients.models import AddressType
 
 deps = ['patient_stage']
 
 
 def load_data(**kwargs):
     load_address_types()
+    load_default_notifications()
 
 
 def load_address_types():
     AddressType.objects.get_or_create(type="Home", description="Home Address")
     AddressType.objects.get_or_create(type="Postal", description="Postal Address")
+
+
+def load_default_notifications():
```

Comment:
Can you please put this in a separate file (as we will have more to come) and just add it to deps on line 12?
Just as `patient_stage` is done...